site/src/components.template

```diff
@@ -0,0 +1,43 @@
+---
+title: Title
+description: The <component-name> component in the Atlas Design System
+template: standard
+figmaEmbed: <figma-url> allowfullscreen
+classType: Component
+classPrefixes:
+  - <component-name>
+---
+
+# Page title (component name)
+
+The first thing is the general description.
+
+Consider using tables to organize information if there are a lot of nuances.
+
+| cssproperty | value                           | screensize |
```

Comment:
This is for atomics, isn't it?

Reply:
Tables can still be used in components docs, so I thought it worth mentioning

- https://design.learn.microsoft.com/components/button.html
- https://design.learn.microsoft.com/components/checkbox.html
- https://design.learn.microsoft.com/components/gradients.html
- https://design.learn.microsoft.com/components/help.html

Reply:
They can, but they won't have the headings `cssproperty | value | screensize`. I would suggest we remove it or at the very least update the headings.

Reply:
Good point, kept the one with ` State | Class | Example ` titles